backend/src/main/java/hu/flowacademy/worksheet/service/UserService.java

```diff
@@ -30,6 +34,16 @@ public User saveUser(@NonNull User user) throws ValidationException {
         return userRepository.save(user);
     }
 
+    public User update(Long id, User user) throws ValidationException {
+        validateUpdate(user);
+        User updatedUser = userRepository.findById(id).orElseThrow(() -> new ValidationException("The id is null: " + user.getId()));
+        log.error("Id is null, or invalid: {}", user.getId());
```

Comment:
Miért az error log?

Reply:
@borosr  Egy korábbi kommentben kérted, hogy egy log.error-t készítsek ide és loggoljam a hibát. "Ez így nem túl biztonságos (NullPointerException gyanús). Kezeld azt is, ha nem található id alapján (orElseThrow(...)), ami dobjon validációs hibát, illetve loggod, hogy milyen id-val történt a hibás lekérés, a log level legyen error." Vagy félreértettem az instrukciót?

Reply:
Oké, máshogy értettem, töröld kérlek ezt a logot, mert így akkor írja a hibaüzenetet, ha minden rendben van.